docs/config-presets-and-options.md

```diff
@@ -0,0 +1,70 @@
+# Configuration Presets and Options
+
+## Defining the Openshift Cluster Topology
+
+Openshift clusters can have different topologies. In Openshift development we refer to 
+these different topologies as different installations with adjectives and acronyms. 
+Below is the list of cluster toplogies currently being used to test the installers:
+
+|                       | Master Nodes | Worker Nodes | Masters Schedulable |
+|-----------------------|--------------|--------------|---------------------|
+| High Availablity (HA) |       3+     |       2+     |         No          |
+| Compact               |       3      |       0      |         Yes         |
+| Single Node Openshift |       1      |       0      |         Yes         |
+
+
+### Resources Used for Openshift Installation Validations in the Lab
+
+|                     | High Availablity (HA) | Compact | Single Node OpenShift (SNO) |
+|---------------------|-----------------------|---------|-----------------------------|
+| NUM_MASTERS         |           3+          |    3    |              1              |
+| NUM_WORKERS         |           2+          |    0    |              0              |
+| NUM_EXTRA_WORKERS   |           0           |    0    |              0              |
+| MASTER_MEMORY       |         16384         |  16384  |            32768            |
+| MASTER_DISK         |           20          |    20   |              20             |
+| MASTER_VCPU         |           8           |    8    |              8              |
+| WORKER_MEMORY       |         16384         |         |                             |
+| WORKER_DISK         |           20          |         |                             |
+| WORKER_VCPU         |           4           |         |                             |
+| EXTRA_WORKER_MEMORY |         16384         |         |                             |
+| EXTRA_WORKER_DISK   |           20          |         |                             |
+| EXTRA_WORKER_VCPU   |           4           |         |                             |
+
+
+### Minimum Required Resources for Production Openshift Clusters
```

Comment:
The minimum memory requirement defined in assisted-service for worker is 8192 and for SNO is 16384: https://github.com/openshift/assisted-service/blob/master/data/default_hw_requirements.json

Reply:
As mentioned before, I'm really not keen about documenting production environments trais in this project, as it does not seem the correct location

Reply:
Yeah I can change the headings for these scenarios. The difference is mainly there are required defaults in assisted-service. These defaults are informed by the official Openshift documentation. However, we override these variables on the regular because most of our lab machines have disks much smaller than 120GB (x5) to make the 5 VMs.